src/tFPDF/PDF.php

```diff
@@ -1431,8 +1431,10 @@ public function Cell($flt_width, $flt_height = 0, $str_text = '', $int_border =
      * @param int $int_border
      * @param string $str_alignment
      * @param bool $bol_fill
+     *
```

Comment:
Thanks for reformatting. Now for the last request, I promise!

Could you please add here
```
@param int int_maxline
```

And I will be happy to merge the changes.